src/server/data_store/lru_cache_data_store.cc

```diff
@@ -0,0 +1,51 @@
+#include "lru_cache_data_store.h"
+
+::std::string KeyValuePair::get_key() const {
```

Comment:
I feel like there's boilerplate code generator that can do this.. they have that for java with @Getter and @Setter annotation.. not sure what's that for c++

Reply:
There's probably library for that but I think we should think really hard on whether it's worth it before introducing new dependency. For now, I think hand-written getters/setters are fine.